aws-py-static-website/__main__.py

```diff
@@ -0,0 +1,166 @@
+import json
+import mimetypes
+import os
+
+from pulumi import export, FileAsset, ResourceOptions, Config, Output
+import pulumi_aws
+import pulumi_aws.acm
+import pulumi_aws.cloudfront
+import pulumi_aws.config
+import pulumi_aws.route53
+import pulumi_aws.s3
+
+def get_domain_and_subdomain(domain):
+    parts = domain.split('.')
+    if len(parts) < 2:
+        raise Exception(f'No TLD found on ${domain}')
+    if len(parts) == 2:
+        return "", domain
+    subdomain = parts[0]
+    parts.pop(0)
+    return subdomain, '.'.join(parts) + '.'
+
+stack_config = Config('static-website')
```

Comment:
This source file would benefit a lot from having some comments. Personally having idiomatic doc comments on the functions would be great, but at the very least having comments to help organize the code would make this a little less intimidating.

For example, something like:

```python
# Read the stack's configuration data.
stack_config = Config('static-website')
...
```